src/components/Header.js

```diff
@@ -1,10 +1,64 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ user, profileImg, changeUser }) {
+  const handleButtonClick = () => {
+    changeUser();
+  };
+  return (
+    <Wrapper>
+      <button onClick={handleButtonClick}>
```

Comment:
`onClick`에 바로 받아온 `changeUser`를 주지 않고, 함수를 또 만든 이유가있을까요?? 궁금해요 ㅎㅅㅎ

```suggestion

  return (
    <Wrapper>
      <button onClick={changeUser}>
```

Reply:
오... 원래 그 함수내에 사진과 이름을 바꿔야 되는줄 알았는데 리렌더링되면서  그럴필요가 없어졌네요!! 감사합니다